hikari/internal/time.py

```diff
@@ -134,7 +136,7 @@ def datetime_to_discord_epoch(timestamp: datetime.datetime) -> int:
     builtins.int
         Number of milliseconds since `1/1/2015 00:00:00 UTC`.
     """
-    return int((timestamp.timestamp() - DISCORD_EPOCH) * 1_000)
+    return int((timestamp - DISCORD_EPOCH).total_seconds() * 1_000)
```

Comment:
Did you consider just using timedeltas for these

Reply:
Changed to it :+1: